### PR TITLE
Rockets basic structure

### PR DESCRIPTION
**Pull request by Lusindiso**
### In this PR I have made the following changes:
- Create a store, with configure store,
- Connect the store with Provider in the ```index.js```
- Create a slice for the rocket reducer.

### CODE REVIEW

File: src/redux/rockets/rocketsSlice.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+});
+
+export const rocketsActions = rocketsSlice.actions;
+
+export default rocketsSlice;
```
Comment (myelin0) [inline src/redux/rockets/rocketsSlice.js:15]: I think what you want to export at the end is "export default rocketsSlice.reducer".The actions you export in line 13. You can change line 15 to export the reducer to the store.

**Review by myelin0 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.

**Comment by Lusindiso** [comment]
@myelin0 
Thank you have reviewing my code. I have implemented the recommended changes.

File: src/redux/rockets/rocketsSlice.js (hunk 1 above)
Comment (Lusindiso) [reply src/redux/rockets/rocketsSlice.js:15]: @myelin0 
Thank you for this suggestion, and I  have called the rocketsSlice.reducer in the ```index.js``` file when setting up the reducers.

**Review by myelin0 — APPROVED** [review]
## STATUS: APPROVED 🟢
## Hi @Lusindiso 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**